components.html

```diff
@@ -36,6 +36,38 @@ <h2 class="component__name">Logo</h2>
               </a>
             </div>
           </li>
+          <li class="component">
+            <h2 class="component__name">Button</h2>
+            <p class="component__description">Buttons are used on pages form and catalog.</p>
+            <p class="component__description">
+              Each button must use the base class "button" and the "view" key modifier. Available options for "view" key modifier:
+              <ul>
+                <li>button_view_primary</li>
+                <li>button_view_secondary</li>
+                <li>button_view_tertiary</li>
+              </ul>
+            </p>
+            <p class="component__description">By default, the button has a width of 100%. It can be changed by using the "place" key modifiers ("button_place_form" or "button_place_photo-editor") or the layout of the parent element (for example, padding).</p>
```

Comment:
@MarPostovik I appreciate the thoughtful use of list and list item tags to enhance the structure of the UI component description. It definitely helps to make it clearer and more organized!

However, if we try to run our HTML through [Nu HTML Checker](https://validator.w3.org/nu/) that will indicate an error:
<img width="462" alt="Screenshot 2024-02-01 at 2 10 20 AM" src="https://github.com/p-foundation/pink-app/assets/76534205/6cee6685-5e91-432d-8605-7a76bc1c540f">


What it really is that we should avoid using `<ul>` inside of inline-level elements. According to [HTML specifications](https://www.w3.org/TR/html401/struct/text.html#h-9.3.1), 

> `<p>` element cannot contain block-level elements(including `<p>` itself). 

While modern browsers might still render it correctly, it is advised to follow semantic HTML guidelines for improved accessibility and code maintainability.

The good news is, there's an easy fix. We should just move the list with all its items after the paragraph.

Reply:
@MarPostovik Let's also add few more examples on how we can control the width of the button by its parent. 
I suggest adding this line: 

> .. (for instance, leveraging CSS Flexbox or CSS Grid capabilities, adjusting parent paddings, etc.).